tutorials/scripts/02_Intro_to_DBO/ackley.py

```diff
@@ -0,0 +1,25 @@
+import time
+import numpy as np
+from deephyper.problem import HpProblem
+from deephyper.evaluator import profile
+
+d = 10
+domain = (-32.768, 32.768)
+hp_problem = HpProblem()
+for i in range(d):
+    hp_problem.add_hyperparameter(domain, f"x{i}")
+
+def ackley(x, a=20, b=0.2, c=2*np.pi):
+    d = len(x)
+    s1 = np.sum(x ** 2)
+    s2 = np.sum(np.cos(c * x))
+    term1 = -a * np.exp(-b * np.sqrt(s1 / d))
+    term2 = -np.exp(s2 / d)
+    y = term1 + term2 + a + np.exp(1)
+    return y
+
+@profile
```

Comment:
why is the decorator here? it does not look like it is being used somewhere.
pack the whole code in a single file, use `.. litteralinclude` to reference the code.